osidb/filters.py

```diff
@@ -779,3 +780,16 @@ class Meta:
             # versions fields
             "versions__version": ["exact"],
         }
+
+
+class AlertFilter(IncludeFieldsFilterSet, ExcludeFieldsFilterSet):
+    owner_uuid = CharFilter(field_name="object_id")
+    owner_model = CharFilter(field_name="content_type__model")
```

Comment:
Suggestion: This might be turned into a ChoiceFilter, I think we can retrieve all models which are subclassing some mixin like this approx:
```python
In [32]: def get_mixin_subclases(mixin):
    ...:     result = []
    ...:     for subclass in mixin.__subclasses__():
    ...:         if subclass._meta.abstract:
    ...:             result.extend(get_mixin_subclases(subclass))
    ...:         else:
    ...:             result.append(subclass.__name__)
    ...:     return result
    ...:

In [33]: get_mixin_subclases(AlertMixin)
Out[33]:
['Flaw',
 'Affect',
 'FlawCVSS',
 'AffectCVSS',
 'FlawComment',
 'FlawAcknowledgment',
 'FlawReference',
 'Package',
 'Snippet',
 'Tracker',
 'FlawMeta']
```

Reply:
To avoid computational overhead (if the subclasses have to be recomputed each time we get the choices) I have just added the possible options in the OpenApiParameter definition; this way we also get a list of options in the docs, although they will have to be revised in the event that the architecture changes.

Reply:
Sure, that is a good idea. That little function I included might go into `osidb/helpers.py` and it can be used to detect any changes from dev point of view. We already have some similar helper functions there as well (eg. get_unique_meta_attr_keys).

Reply:
I will add it there. I did use a pretty similar function to get the list of models for the migration, but it was done ad-hoc,  didn't think about keeping it somewhere for future reference.